apache/netbeans
# Update asm from 9.4 to 9.5

Library Notes:
- Support for Java 21
- New readBytecodeInstructionOffset hook in ClassReader
- More detailed exception messages
- Javadoc improvements and fixes
- Bug fixes:
  - 317989: Silent removal of zero-valued entries from the line-number table


Testing:
- Full build done
- Verify successful execution of libraries and licenses Ant test
- Verify successful execution of `ant check-sigtests`
- Verify that the jar is located in `netbeans_dir/platform/core/asm-9.5.jar`

[ASM Web Page](https://asm.ow2.io/index.html)
[Release Notes](https://asm.ow2.io/versions.html)
